tests/validation/tests/v3_api/test_ingress.py

```diff
@@ -321,6 +366,132 @@ def test_ingress_xip_io():
     validate_ingress_using_endpoint(namespace["p_client"], ingress, [workload])
 
 
+@if_test_rbac
+@pytest.mark.parametrize("role", rbac_role_list)
+def test_rbac_create_ingress(role):
```

Comment:
Can this be renamed to test_rbac_ingress_create? Keep this naming convention for the other rbac cases as well.

Reply:
Done